packages/graphql/lib/federation/graphql-federation-definitions.factory.ts

```diff
@@ -35,9 +36,17 @@ export class GraphQLFederationDefinitionsFactory extends GraphQLDefinitionsFacto
         resolvers: {},
       },
     ]);
+
+    const mergedDefinition = mergeTypeDefs([printSubgraphSchema(schema)], {
```

Comment:
Should we add a comment explaining why this is needed? (with a link to the original issue)